contrib/runtime_metrics/src/main/java/io/opentelemetry/contrib/metrics/runtime/MemoryPools.java

```diff
@@ -116,14 +113,14 @@ public void exportMemoryPoolMetric() {
             .setLabelKeys(Arrays.asList(TYPE_LABEL_KEY, POOL_LABEL_KEY))
             .setMonotonic(false)
             .build();
-    final List<LabelSet> usedLabelSets = new ArrayList<>(poolBeans.size());
-    final List<LabelSet> committedLabelSets = new ArrayList<>(poolBeans.size());
-    final List<LabelSet> maxLabelSets = new ArrayList<>(poolBeans.size());
+    final List<String[]> usedLabelSets = new ArrayList<>(poolBeans.size());
+    final List<String[]> committedLabelSets = new ArrayList<>(poolBeans.size());
+    final List<String[]> maxLabelSets = new ArrayList<>(poolBeans.size());
     for (final MemoryPoolMXBean pool : poolBeans) {
-      usedLabelSets.add(meter.createLabelSet(TYPE_LABEL_KEY, USED, POOL_LABEL_KEY, pool.getName()));
+      usedLabelSets.add(new String[] {TYPE_LABEL_KEY, USED, POOL_LABEL_KEY, pool.getName()});
       committedLabelSets.add(
-          meter.createLabelSet(TYPE_LABEL_KEY, COMMITTED, POOL_LABEL_KEY, pool.getName()));
-      maxLabelSets.add(meter.createLabelSet(TYPE_LABEL_KEY, MAX, POOL_LABEL_KEY, pool.getName()));
+          new String[] {TYPE_LABEL_KEY, COMMITTED, POOL_LABEL_KEY, pool.getName()});
+      maxLabelSets.add(new String[] {TYPE_LABEL_KEY, MAX, POOL_LABEL_KEY, pool.getName()});
```

Comment:
Can you call a vararg with an array?

Reply:
yep! varargs are just some syntactic sugar for an array.